newsletter-june-2021.rst

```diff
@@ -0,0 +1,88 @@
+.. post:: June 3, 2021
+   :tags: newsletter, python
+   :author: Juan Luis
+   :location: MAD
+
+.. meta::
+   :description lang=en:
+      Company updates and new features from last month,
+      current focus, and upcoming features from June.
+
+Read the Docs newsletter - June 2021
+====================================
+
+Welcome to a new edition of our monthly newsletter, where we
+openly share the most relevant updates around Read the Docs,
+offer a summary of new features we shipped
+during the previous month,
+and share what we'll be focusing on in the near future.
+
+Company highlights
+------------------
+
+- We applied to the next round of the CZI Essential Open Source Software for Science!
+  Our proposal is even more ambitious than the previous one,
+  and we are excited about the possibilities for our users
+  if we finally get accepted.
+- We keep making progress on the current CZI grant work:
+  `Juan Luis`_ sent the first pull request for `our new Sphinx
+  tutorial <https://github.com/sphinx-doc/sphinx/pull/9276>`_,
+  which received positive feedback from the maintainers,
+  and `Manuel`_ is about to release a new version of ``sphinx-hoverxref``
+  `with support for intersphinx <https://github.com/readthedocs/sphinx-hoverxref/pull/86>`_
+  (we will write a dedicated blog post about this soon).
+- We added a `Data Processing Agreement <https://docs.readthedocs.io/en/stable/legal/dpa/>`_
+  to our official documentation.
+- We are making progress towards a new streamlined workflow
+  that helps us while the company grows.
+
+.. Pageviews stats obtained from Google Analytics, https://readthedocs.io property,
+   and divided by the total number of days in the month
+
+New features
+------------
+
+- We added a new configuration key, ``build.apt_packages``,
+  that allow our users to install custom operating system packages.
+  You can read more `in our dedicated blog post </apt-packages>`_.
+- Now redirects also work `even if you are using
+  HTTP <https://github.com/readthedocs/readthedocs.org/issues/8183>`_.
+- Our API v2 now allows `filtering more
+  endpoints <https://github.com/readthedocs/readthedocs.org/pull/8189>`_.
```

Comment:
This was also a fix from a regression that happened that week p: we were already allowing those filters

Reply:
We don't have a dedicated section in the newsletter called "bug fixes", so I liberally decided to include them in "new features" 😅 I don't know if it's worth separating them. Although definitely I can rephrase them so they properly reflect the reality.

Reply:
Yeah, this particular case is not a feature and probably we don't want to communicate this to users. Basically, we are blocking list requests to resources of APIv2 to avoid people taking the site down.

Reply:
Fair enough! Removing it